keras/backend/tensorflow_backend.py

```diff
@@ -2255,17 +2286,24 @@ def __init__(self, inputs, outputs, updates=None, name=None, **session_kwargs):
     def __call__(self, inputs):
         if not isinstance(inputs, (list, tuple)):
             raise TypeError('`inputs` should be a list or tuple.')
-        feed_dict = {}
+        self.current_feed_dict = self.feed_dict
+        if self.feed_dict is None:
+            self.current_feed_dict = {}
```

Comment:
self.current_feed_dict = {} if self.feed_dict is None else self.feed_dict

Clearer